.github/workflows/genlangdb.yml

```diff
@@ -0,0 +1,68 @@
+name: Generate wxWidgets language database
+
+on: workflow_dispatch
```

Comment:
Well, it does no harm to run this workflow regularly. But I don't see much advantage in doing so. Underlying Unicode or Windows data files maybe change once per year, maybe even less often. If the GitHub action breaks (for example, due to an outdated action), it will have to be fixed, of course. However, it will not be an urgent task - whether some more or less exotic languages are supported a bit earlier or later shouldn't matter too much.